Installation/changes.html

```diff
@@ -179,6 +179,19 @@ <h2 id="release4.9">Release 4.9 </h2>
 <!-- Mesh Generation -->
 <!-- Surface Reconstruction -->
 <!-- Geometry Processing -->
+<h3>Random Point Generators</h3>
+<ul>
+  <li>
+    Add point random generators
+    <ul>
+    <li>in a 3D triangle mesh model of the concept <code>FaceListGraph</code> (<code>CGAL::Random_points_in_triangle_mesh_3</code>),</li>
+    <li>on the boundary of a tetrahedral mesh (<code>CGAL::Random_points_in_tetrahedral_mesh_boundary_3</code>),</li>
+    <li>in a tetrahedral mesh (<code>CGAL::Random_points_in_tetrahedral_mesh_3</code>),</li> 
+    <li>in a 2D triangle mesh (<code>CGAL::Random_points_in_triangle_mesh_2</code>),</li>
+    <li>in a range of 2D or 3D triangles (<code>CGAL::Random_points_in_triangles_3</code> and <code>CGAL::Random_points_in_triangles_2</code>).</li>
+    </ul>
+  </li>
+</ul>
```

Comment:
Please copy the changes.html from `master` (4.10) and move this change to the right section.